addons/website/tests/test_base_url.py

```diff
@@ -85,7 +85,7 @@ def test_02_canonical_url(self):
         self._assertCanonical('/?debug=1', self.website.get_base_url() + '/')
         self._assertCanonical('/a-page', self.website.get_base_url() + '/a-page')
         self._assertCanonical('/en_US', self.website.get_base_url() + '/')
-        self._assertCanonical('/fr_FR', self.website.get_base_url() + '/fr/')
+        self._assertCanonical('/fr_FR', self.website.get_base_url() + '/fr')
```

Comment:
Maybe we can add those 2 also?
```suggestion
        self._assertCanonical('/fr_FR', self.website.get_base_url() + '/fr')
        self._assertCanonical('/fr_FR/', self.website.get_base_url() + '/fr')
        # (and between L86-87):
        self._assertCanonical('/a-page/', self.website.get_base_url() + '/fr')
```